main.css

```
@@ -0,0 +1,333 @@
+@import "palette.css";
+*{
+  box-sizing : border-box;
+}
+
+body{
+  font-family: Pretendard;
+  display: flex;
+  flex-direction: column; 
+  margin: 0;
+  align-items: center;
+} 
+
+nav {
```

엘리먼트로만 제어하면 사이드 이펙트가 발생할 여지가 있습니다.
전역에서 사용하는 UI인 만큼 명시적인 셀렉터를 넣어주는게 좋을 것 같아요!
```suggestion
nav#gnb {
```